src/libyaml/libyaml_lgpl/COPYING

```diff
@@ -0,0 +1,24 @@
+This software is licensed by RDK Management under the GNU General
+Public License version 2.0 only, as per COPYING.LGPL and subject to any
+additional licenses included in this file.
+
+Please use the following copyright for source under this directory:
+
+Copyright 2023 RDK Management
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Library General Public
+License as published by the Free Software Foundation; either
+version 2 of the License, or (at your option) any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Library General Public License for more details.
+
+You should have received a copy of the GNU Library General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+Copyright (C) 2001       Anil Kumar
```

Comment:
Copyright (c) 2017-2020 Ingy döt Net
Copyright (c) 2006-2016 Kirill Simonov

Original Authors https://github.com/yaml/libyaml?tab=MIT-1-ov-file

Reply:
Added the correct authors:
https://github.com/pantoniou/libfyaml/tree/master?tab=MIT-1-ov-file